### DISCUSSION TITLE
Hide model Id/Connection

### DISCUSSION THREAD

**Discussion by francoism90** [filamentphp/filament#6832] [Help]
See https://github.com/livewire/livewire/discussions/2627 for more details.

One of the things I'm facing with Filament, is the `serverMemo` can be called from JS and basically read model details.
It shows the ID, model location/class and even the connection type (mysql in this case).

Would it be possible to hide this/overrule the reported record id? If one uses already routing binding, it should work right?
Or would it be possible to use https://github.com/spatie/laravel-data, but inside the admin?

Thanks.

**Comment by redrum0x** [comment]
Does anyone have an answer?

**Reply by francoism90** [reply]
@redrum0x Checkout https://github.com/livewire/livewire/pull/6598

This is my workaround: https://github.com/livewire/livewire/pull/6335#issuecomment-1693865407
